CHEATSHEET.md

```diff
@@ -3,25 +3,160 @@
 ## Administration
 
 * Log into Drupal
+[Link to login.js](login.js)
+cy.login('User', 'pass')
+
+* Logout 
+[Link to logout](logout.js)
+cy.logout()
+
 * Go to a page
+cy.visit('/path')
 
 ## Creating a Node
 * Go to a node add form
+cy.visit('/node/add'); // Replace with the actual URL of your content
+
 * Fill out a text field
+cy.get('#id-selector').type(enter-string);
+
 * Choose a select list option
+cy.get('#id-selector').select('option');
+
 * Choose a radio or checkbox
+** In Cypress, you can interact with radio buttons and checkboxes on a Drupal site using the .check() and .click()
+cy.get('#id-selector').click();
+
 * Choose an autocomplete
+** Use Cypress autocomplete command will select first option that appears for autocomplete
+[Link to autocomplete](autocomplete.js)
+cy.autocomplete('[data-drupal-selector="edit-field-sample-0-target-id"]', 'item');
+
 * Fill out a rich text editor field
+** Use ckeditorType command 
+[Link to ckeditortype](ckeditorType.js)
+cy.ckeditorType('#edit-body-0-value', 'string of text')
+
 * Create a Paragraph
```

Comment:
For multiple line code blocks, ass ```

```
cy.visit('/node/123/edit');

cy.get('.add-paragraph-button').click();

cy.get('#edit-paragraph-field-name').type('My Paragraph Name');
// Fill in other paragraph fields here
cy.get('#edit-submit').click();
```